new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,209 @@
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static oracle.weblogic.kubernetes.actions.TestActions.createIngress;
+import static oracle.weblogic.kubernetes.actions.TestActions.createUniqueNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.deleteNamespace;
+import static oracle.weblogic.kubernetes.actions.TestActions.helmGetExpectedValues;
+import static oracle.weblogic.kubernetes.actions.TestActions.helmList;
+import static oracle.weblogic.kubernetes.actions.TestActions.installTraefik;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallIngress;
+import static oracle.weblogic.kubernetes.actions.TestActions.uninstallTraefik;
+import static oracle.weblogic.kubernetes.actions.TestActions.upgradeTraefik;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.traefikIsRunning;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.awaitility.Awaitility.with;
+
+// this is an example of a test suite (class) where the tests need to be run in a certain
+// order. this is controlled with the TestMethodOrder annotation
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("Simple validation of Traefik functions")
+@ExtendWith(Timing.class)
+class ItSimpleTraefikValidation implements LoggedTest {
+
+  private HelmParams tfHelmParams = null;
+  private HelmParams ingressParam = null;
+  private String tfNamespace = null;
+  private String domainNamespace1 = null;
+
+  @Test
+  @Order(1)
+  @DisplayName("Install the Traefik")
+  // tags are used to filter which tests to run, we can define whatever tags we need,
+  // like these two:
+  @Slow
+  @MustNotRunInParallel
+  public void testInstallingTraefik() {
+
+    // get a new unique traefik Namespace
+    tfNamespace = createNamespace();
+    logger.info(String.format("Created a new namespace called %s", tfNamespace));
+
+    String projectRoot = System.getProperty("user.dir");
+
+    // helm install parameters
+    tfHelmParams = new HelmParams()
+        .releaseName("traefik-operator")
+        .namespace(tfNamespace)
+        .repoUrl("https://kubernetes-charts.storage.googleapis.com/")
+        .repoName("stable")
+        .chartName("traefik")
+        .chartValuesFile(projectRoot + "/../kubernetes/samples/charts/traefik/values.yaml");
+
+    TraefikParams tfParams = new TraefikParams()
+        .helmParams(tfHelmParams)
+        .nameSpaces("{" + tfNamespace + "}");
+
+    // install Traefik
+    assertThat(installTraefik(tfParams))
+        .as("Test installTraefik returns true")
+        .withFailMessage("installTraefik() did not return true")
+        .isTrue();
+    logger.info(String.format("Traefik is installed in namespace %s", tfNamespace));
```

Comment:
Same comment for all usages of String.format in logger calls.

Reply:
done